src/dialogs/linkdialog.ui

```diff
@@ -11,7 +11,7 @@
    </rect>
   </property>
   <property name="windowTitle">
-   <string>Search note</string>
```

Comment:
Because I thought that was a bad copy/paste or something like that because "Search note" doesn't seam to be good for the link dialog (where we can also link to an URL)